### PR TITLE
Remove Unused Parameter Warnings

### PR DESCRIPTION
**Pull request by jmgate**
@trilinos/ifpack, @trilinos/zoltan, @trilinos/zoltan2 

## Description
Removing unused parameter warnings generated by `gcc-7.3.0` by commenting out parameter names in function definitions.

## Motivation and Context
Just hoping for a clean build.

## Related Issues
* Analogous to #4256, #4243, #4239.

## How Has This Been Tested?
Everything still builds on my RHEL7 machine.  Letting the @trilinos-autotester do its thing.

### CODE REVIEW

File: packages/zoltan2/src/problems/Zoltan2_Problem.hpp [hunk 1]
```diff
@@ -235,7 +235,7 @@ class Problem : public ProblemRoot {
 };
 
 template <typename Adapter>
-  void Problem<Adapter>::setupProblemEnvironment(ParameterList *params)
+  void Problem<Adapter>::setupProblemEnvironment(ParameterList * /* params */)
```
Comment (kddevin) [inline packages/zoltan2/src/problems/Zoltan2_Problem.hpp]: @kddevin should look more closely at this one; perhaps the argument isn't needed.

**Comment by jmgate** [comment]
Thanks @kddevin.  I removed the "AT: AUTOMERGE" label so the testing will complete but the PR won't get merged without someone clicking the merge button.

**Comment by jmgate** [comment]
@kddevin, ping&hellip;

**Comment by jmgate** [comment]
Well shucks.  I'm afraid I don't have time to debug this now.  Closing.  I'll wind up submitting a similar PR some time in the future when I have some free time.

**Comment by mhoemmen** [comment]
@jmgate Intel license server is down, which is hindering PR testing.  Not sure if related.

**Pull request closed without merging**